tests/test_rbuser_tostring.py

```diff
@@ -1,80 +1,33 @@
-                                'loginShell': '/usr/local/shells/zsh',
-                                'objectClass': ['redbrick', 'posixAccount', 'top', 'shadowAccount'],
-                                'oldusertype': 'newb',
-                                'userPassword': 'e1NTSEF9WlBURGtJdmZJSld0WUlrak9zdGxscXYySTRRVEhj=',
-                                'homeDirectory': '/home/redbrick/memberOne',
-                                'disuser_period': 0,
-                                'shadowLastChange': 17073, }
-        self.member_one_output = \
-            'cn                :  Member One\n' + \
-            'id                :  15358462\n' + \
-            'uid               :  memberOne\n' + \
-            'bday              :  12\n' + \
-            'host              :  carbon\n' + \
-            'year              :  3\n' + \
-            'byear             :  2017\n' + \
-            'gecos             :  new member\n' + \
-            'bmonth            :  2\n' + \
-            'course            :  CASE\n' + \
-            'newbie            :  True\n' + \
-            'passwd            :  e1NTSEF9WlBURGtJdmZJSld0WUlrak9zdGxscXYySTRRVEhBYj=\n' + \
-            'altmail           :  memberOne@example.dcu.ie\n' + \
-            'created           :  2009-10-06 11:20:13\n' + \
-            'updated           :  2015-09-28 01:42:21\n' + \
-            'birthday          :  12 02 2017\n' + \
-            'usertype          :  member\n' + \
-            'createdby         :  admin\n' + \
-            'gidNumber         :  1017\n' + \
-            'uidNumber         :  102007\n' + \
-            'updatedby         :  admin\n' + \
-            'yearsPaid         :  1\n' + \
-            'loginShell        :  /usr/local/shells/zsh\n' + \
-            'objectClass       :  [\'redbrick\', \'posixAccount\', \'top\', ' + \
-            '\'shadowAccount\']\n' + \
-            'oldusertype       :  newb\n' + \
-            'userPassword      :  e1NTSEF9WlBURGtJdmZJSld0WUlrak9zdGxscXYySTRRVEhj=\n' + \
-            'homeDirectory     :  /home/redbrick/memberOne\n' + \
-            'disuser_period    :  0\n' + \
-            'shadowLastChange  :  17073\n'
-
-    def test_to_string(self):
-        """Test case A. note that all test method names must begin with 'test.'"""
+    def test_member_one(self):
+        """Test case A. Member One"""
         member_one = rbuser.RBUser()
-        member_one.set_attr(**self.member_one_attr)
-        assert member_one.__str__() == self.member_one_output, 'member1 attributes not as expected'
+        member_one.set_attr(**data.MEMBER_ONE_ATTR)
+        to_string_output = member_one.__str__()
+        assert to_string_output == data.MEMBER_ONE_OUTPUT, \
+            'member1 attributes not as expected. ' + \
+            'Diff at: ' + [i for i in range(len(data.MEMBER_ONE_OUTPUT))
+                           if data.MEMBER_ONE_OUPUT[i] != to_string_output[i]]
+
+    def test_associate_one(self):
```

Comment:
No, I tried that but pyunit runs all functions beginning with test and that is the out that you see.

So to see what test fails and fully utilise pyunit this is the way to do it.